### ISSUE TITLE
A problem with usage

### ISSUE THREAD

**Issue by egorsmkv** [MiniXC/phones#13]
I have install the library using the `pip install phones` command and ran the following script:

```
from phones.convert import Converter

converter = Converter()
```

but have this error:

```
Traceback (most recent call last):
  File "ipa_arpa.py", line 3, in <module>
    from phones.convert import Converter
  File "/Users/yehorsmoliakov/opt/miniconda3/lib/python3.8/site-packages/phones/convert.py", line 22, in <module>
    from .phonecodes.src import phonecodes
ModuleNotFoundError: No module named 'phones.phonecodes'
```

**Comment by egorsmkv** [comment]
and there is an artifact after merging - https://github.com/MiniXC/phonecodes/blob/master/src/phonecodes.py#L24

**Comment by MiniXC** [comment]
Was a build problem since I forgot to clone submodules for the last patch. This is fixed now (0.0.6).
Highlights the need for some basic tests at the very least though.
Also removed the merge artifact, thanks for reporting this!

**Comment by egorsmkv** [comment]
Thanks Sent from my iPhoneOn 5 Jan 2023, at 13:00, Christoph Minixhofer ***@***.***> wrote:﻿
Closed #13 as completed.
